ecms_base/features/custom/ecms_publications/config/install/core.base_field_override.paragraph.publication_list.created.yml

```diff
@@ -0,0 +1,17 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - paragraphs.paragraphs_type.publication_list
+id: paragraph.publication_list.created
+field_name: created
+entity_type: paragraph
+bundle: publication_list
+label: 'Authored on'
+description: 'The time that the Paragraph was created.'
+required: false
+translatable: true
```

Comment:
I think this should be false. I checked the other created fields and they are all false.